---
params:
  species: "Brown bear"
  years: !r 2012:2024
  date: !r Sys.Date()   
  working.dir: !r getwd()
  out: !r list()
title: "`r paste('Estimates of', params$species, 'density, abundance, and population dynamics in Scandinavia,', params$years[1], '-', params$years[length(params$years)]')`"
output:
  pdf_document: default
  html_document: 
    theme: "cerulean" 
    toc_depth: 2
    toc: yes
    toc_float: yes
    df_print: kable
    # css: bootstrap.css
    highlight: zenburn
    # fig_caption: true
  word_document: default
bibliography: references.bib
---

```{r, setup, echo = FALSE, warning = FALSE, message = FALSE}
years <- params$years
n.years <- length(years)
species <- params$species
working.dir <- params$working.dir
date <- params$date

##-- Species
if(sum(grep("bear", species, ignore.case = T))>0|sum(grep("bjorn", species, ignore.case = T))>0){
  engSpecies <- "bear"
  EngSpecies <- "Bear"
  latSpecies <- "Ursus arctos"
  species <- "Brown bear"
}
if(sum(grep("wolf", species, ignore.case = T))>0|sum(grep("ulv", species, ignore.case = T))>0){
  engSpecies <- "wolf"
  EngSpecies <- "wolf"
  latSpecies <- "Canis lupus"
  species <- "Gray wolf"
}
if(sum(grep("wolverine", species, ignore.case = T))>0|sum(grep("jerv", species, ignore.case = T))>0){
  engSpecies <- "wolverine"
  EngSpecies <- "Wolverine"
  latSpecies <- "Gulo gulo"
  species <- "Wolverine"
}

##-- Months list
months = c("January","February","March","April","May","June",
           "July","August","September","October","November","December")

##-- Libraries
library(kableExtra)
library(rovquantR)
```

```{r, data, echo = FALSE}
##-- Load pre-processed habitat shapefiles
data(COUNTRIESWaterHumans, envir = environment()) 
COUNTRIES <- COUNTRIESWaterHumans[COUNTRIESWaterHumans$ISO %in% c("SWE","NOR"), ] %>%
  dplyr::group_by(ISO) %>%
  dplyr::summarise()

##-- Get extraction date
fileList <- list.files(file.path(working.dir, "data")) 
fileName <- fileList[grep(paste0("CleanData_",engSpecies,"_"), fileList)]
extractDate <- sub("^.*_(.*)\\.RData$", "\\1",  fileName)

##-- Load detectors
load(file.path( working.dir, "data", paste0("Detectors_", engSpecies, "_", date, ".RData")))

##-- Load results tables
N_LastYear <- read.csv(file.path( working.dir, "tables/Abundance_LastYearPerSex.csv"), check.names = F, row.names = 1)
N_All <- read.csv(file.path( working.dir, "tables/Abundance_AllYears.csv"),check.names = F, row.names = 1)

##-- Extract abundances for the last year
CI <- as.numeric(strsplit(sub('.*\\((.*)\\).*', '\\1', N_LastYear["Total","Total"]), "-")[[1]])
CI_F <- as.numeric(strsplit(sub('.*\\((.*)\\).*', '\\1', N_LastYear["Total","Females"]), "-")[[1]])
CI_M <- as.numeric(strsplit(sub('.*\\((.*)\\).*', '\\1', N_LastYear["Total","Males"]), "-")[[1]])
```

### `r Sys.Date()`

# Summary

**Background** The Scandinavian `r species` (*`r latSpecies`*) population is monitored annually using non-invasive genetic sampling (NGS) and recovery of dead individuals. DNA extracted from faeces, urine, hair, and tissue is used to identify the species, sex and individual from which each sample originated. These data are compiled annually in the Scandinavian large carnivore database [Rovbase 3.0](https://www.rovbase.no/).

**Approach** Using the Bayesian open-population spatial capture-recapture (OPSCR) model developed by [RovQuant] (<https://www.nmbu.no/en/research/projects/rovquant>), we estimated the status and dynamics of the Scandinavian `r engSpecies` population between `r years[1]` to `r years[length(years)]`. We provide annual density maps, as well as estimates of jurisdiction-specific population sizes, cause-specific survival, recruitment, and detection probabilities. Associated uncertainties are reported with all estimates.

**Results** We estimated that, within its primary range (`r detectors$n.detectors*(detectors$resolution/1000)^2` $km^{2}$), the Norwegian brown bear population was likely (95% credible interval) made up of between `r CI[1]` and `r CI[2]` individuals (95% credible interval) in `r years[length(years)]`, with between `r CI_F[1]` and `r CI_F[2]` females and between `r CI_M[1]` and `r CI_M[2]` males. Each year, a large proportion of the population is detected during annual large carnivore monitoring. Specifically, in `r years[length(years)]`, between `r XXX`% and `r XXX`% of the population were detected during non-invasive genetic sampling.

```{r, summary figure, fig.align = "center", echo = FALSE}
knitr::include_graphics(file.path(working.dir, "figures/UD_Density_summary.png"))
```

------------------------------------------------------------------------

# Introduction

Non-invasive genetic sampling (NGS) and dead recoveries are a centerpiece of national and regional large carnivore monitoring in Norway and Sweden. Both countries have collected an extensive individual-based data set for the `r species` (*`r latSpecies`*), publicly available and stored in the Scandinavian large carnivore database Rovbase ([rovbase.no](https://www.rovbase.no/), [rovbase.se](https://www.rovbase.se/)).

Since 2017, project [RovQuant](https://www.nmbu.no/en/projects/rovquant), from the Norwegian University of Life Sciences (NMBU), has been developing statistical methods to exploit the potential of this database and allow a comprehensive assessment of the population status and dynamics of three large carnivore species, namely the brown bear (*Ursus arctos*), wolf (*Canis lupus*), and wolverine (*Gulo gulo*).

At the core of this analytical framework [@bischof2019rovquant; @bischof2020estimating], are Bayesian spatial capture-recapture (SCR) and open-population spatial capture-recapture (OPSCR) models [@ergon2014; @bischof2016; @chandler2018characterizing; @dupont2021deadreco]. These models use the spatial and temporal information contained in the repeated genetic detections of individuals to estimate various population parameters, including spatially explicit abundance and vital rates. In addition, OPSCR models use the temporal information contained in data collected over multiple years to also estimate recruitment and survival probabilities, as well as inter-annual movements. Importantly, the approach accounts for imperfect detection during sampling (i.e., the fact that some individuals are not detected at all) and animal movement (i.e., the fact that individuals may use and be detected in multiple management units or countries). The OPSCR method brings along several advantages, including the ability to map density, derive jurisdiction-specific abundance, estimate survival and recruitment (which are needed for making population projections), and yield tractable measures of uncertainty [@bischof2019rovquant; @bischof2020estimating] .

RovQuant reported abundance estimates for wolverines and wolves on an annual basis since 2019 [@bischof2019wolf; @bischof2019; @bischof2020estimating; @flagstad2021; @milleret2021wolf; @milleret2021wolverine; @milleret2022wolf; @milleret2022wolverine; @milleret2023wolf; @milleret2023wolverine; @milleret2024wolf] and for brown bears in Norway since 2022 [@dupontNorBear2021; @dupontNorBear2022; @dupontSweBear2023]. During these and other analyses [@milleret2018pab; @milleret2019local; @bischof2020consequences; @dupont2021deadreco; @turek2021efficient; @dey2022; @moqanaki2021], RovQuant has continuously improved the performance of the OPSCR models. In the present report we summarize the analysis of a `r length(years)`-year time-series (`r years[1]`- `r years[length(years)]`) of the latest available `r engSpecies` monitoring data and the most recent version of the OPSCR model. We provide the following information:

-   Annual and sex-specific estimates of the number of `r engSpecies` within the main population range (with credible intervals), as well as estimates by county and large carnivore management region.

-   Annual maps of `r engSpecies` density throughout the species’ primary range in Scandinavia.

-   Annual estimates of survival, cause-specific mortality, recruitment, and population growth rate (with credible intervals).

-   Estimated proportion of individuals detected through non-invasive genetic sampling (with credible intervals).

`r if(engSpecies == “bear”){ paste(“Although RovQuant aims to provide comprehensive analyses of large carnivore populations throughout their entire range in Norway and Sweden, we only analyzed and present results for the Norwegian monitoring data in this report. Previous analyses revealed multiple difficulties related to the Swedish brown bear non-invasive genetic monitoring data (Dupont et al., 2023a; Milleret et al., 2024a), leading among other things to inconsistent results across model permutations when the Scandinavian brown bear monitoring data were analyzed jointly for Norway and Sweden (Dupont et al., 2023b).”)}`

------------------------------------------------------------------------

# Methods

## Data

We included data from multiple sources, the primary one being the Scandinavian large carnivore database Rovbase 3.0 (rovbase.se and rovbase.no; last extraction: `r`). This database is used jointly by Norway and Sweden to record detailed information associated with large carnivore monitoring, including, but not limited to, non-invasive genetic sampling (NGS) data, dead recoveries, and GPS search tracks. In the following sections, we describe the various types of data used in the analysis. We used data collected during winters 2014/2015 to 2023/2024.

**Non-invasive genetic sampling** Swedish and Norwegian management authorities, SNO (Norway) and Länsstyrelserna (Sweden) conduct annual searches for sources of DNA (primarily scats and urine) throughout the Scandinavian `r engSpecies` range [@svensson2023]. Although samples may be collected throughout the year, the official survey period starting with the 2014/2015 season has been Oct 1 – Mar 31. About one third (30%) of DNA samples originated from opportunistic searches (without associated information on search effort) conducted by hunters or other members of the public (see @liberg2012 and @bischof2019rovquant for further details about the data collection procedure). For individual identification, at least 17 microsatellite markers were amplified with PCR throughout the study period. Samples were amplified four times to account for the occurrence of allelic dropout and false alleles [@taberlet1996]. Consensus genotypes were constructed from the replicated PCR runs using the threshold rule that the same alleles had to appear at least twice for a heterozygous genotype and three times for a homozygous genotype [@akesson2016]. Since 2017/2018, 90 single nucleotide polymorphisms (SNPs) were used for individual identification, in addition to microsatellites. Samples were amplified and genotypes visualized twice using nanofluid arrays (Fluidigm Inc.). Consensus SNP genotypes consisted of the markers with the same allele composition from the two amplifications [@akesson2018].

**Dead recoveries** In Scandinavia, all dead large carnivores killed legally (e.g., legal hunting, management kills, defense of life and property) have to be reported to the state authorities (Statsforvalteren or SNO in Norway and Länsstyrelserna or the police in Sweden). All wolves found dead due to other reasons (e.g., natural deaths, vehicle and train collisions, illegal hunting) also have to be reported, but an unknown proportion remains undetected. Tissue is collected from all reported dead carnivores for DNA extraction and analysis. DNA from dead carnivores, if genotyped, can be linked with NGS data via individual IDs and provide definite information about the fate of individuals. Dead recoveries associated with GPS-coordinates also provide additional information for the estimation of individual locations, especially valuable for individuals with few or no NGS detections [@dupont2021deadreco].

**GPS search tracks** Government employees involved in systematic searches for `r engSpecies` DNA along roads and following `r engSpecies` tracks (e.g., via snowmobiles, skis, snowshoes) documented their effort with GPS track logs which were registered in Rovbase 3.0. GPS search tracks were included in the OPSCR model to account for spatial and temporal variation in search effort.

**Observation reports in Skandobs** We used all observation records available from Skandobs (skandobs.se; skandobs.no) that were recorded during the `r engSpecies` monitoring seasons since Oct 1, 2014, (last extraction: 2024-04-08). Skandobs is a web application that allows anyone to anonymously register observations (e.g., visual, tracks, faeces) of bears, lynx, wolves, and wolverines in Scandinavia. This data consists of more than 65 000 records of possible large carnivore observations. Although most observations are not verified, they offer the best available proxy for spatio-temporal variation in opportunistic search effort.

**Parentage analysis and social status assignation** Each year, field observations, genetic sampling data, dead recoveries, and parentage analysis (which resulted in a near-complete pedigree of the Scandinavian wolf population) are used to establish counts of pairs, packs, and reproductions [@svensson2014 @svensson2019 @svensson2021 @svensson2023 @svensson2025 @wabakken2020]. The social status (adult scent-marking member in a pack/pair, or subadult member of a pack and vagrant individuals) was determined on a yearly basis for genetically identified individuals using the pedigree and tracking information from the monitoring @akesson2022. The identity of known adult scent-marking individuals was reported in the yearly monitoring reports [@svensson2014; @svensson2015; @svensson2017; @svensson2019; @svensson2021; @svensson2023; @wabakken2016; @wabakken2018; @wabakken2020; @wabakken2022; @wabakken2024] and a detailed description of the method used for parental assignment was provided in @akesson2016. We assigned individuals to two different states depending on whether the individual had been identified as an adult scent-marking member of a pair or a pack (“adult scent-marking”) or not (“other”). See @bischof2019rovquant for further details.

**Study area** Although `r species` can be occasionally present within the entire Scandinavian range, we restricted the analysis to the core area (Figure 1). Following the recent `r engSpecies` range expansion towards the south of Sweden, the Swedish Environmental Protection Agency requested an expansion of the study area [@milleret2022wolf, @milleret2023wolf]. Since 2022, the RovQuant models therefore included an additional 68 400 km2 in the study area (total= 322 400 km2), compared to the previous analyses of the Scandinavian monitoring `r engSpecies` data by RovQuant [@bischof2019wolf; @bischof2019rovquant; @milleret2021wolf].

## Open-population spatial capture-recapture model

We analysed the data using a Bayesian open-population spatial capture-recapture (OPSCR) model [@bischof2020estimating], which addresses three challenges associated with population-level wildlife inventories:

1.  Detection is imperfect and sampling effort heterogeneous in space and time: not all present in the study area are detected [@kery2011].

2.  Individuals that reside primarily outside the surveyed area may be detected within it. Without an explicit link between the population size parameter and geographic space or area, density cannot be estimated and population size is ill-defined [@efford2004].

3.  Non-spatial population dynamic models usually estimate “apparent” survival and recruitment, as these parameters include the probability of permanent emigration and immigration, respectively. By explicitly modelling movement of individuals between years, the OPSCR model can help return unbiased estimates of demographic parameters [@ergon2014; @schaub2014estimating; @gardner2018; @dupont2022efficient].

The OPSCR model [@ergon2014separating; @bischof2016; @chandler2018characterizing] is composed of three sub-models:

1.  A model for population dynamics and population size.

2.  A model for density and movements.

3.  A model for detections during DNA searches.

**Population dynamics sub-model** We used a multi-state formulation [@lebreton2002multistate] where each individual life history is represented by a succession of up to five discrete states zi,t: 1) “unborn” if the individual has not been recruited in the population (state "unborn" is required for the data augmentation procedure, see below); 2) “alive other” if it is alive and not assigned as an "adult scent-marking" individual; 3) “alive adult scent marking” if it is alive and assigned as an "adult scent-marking" individual; 4) “culled” if it was culled and therefore recovered dead between the start of the previous and current monitoring season; or 5) “dead”: if it has died but was not recovered dead. We then modelled the transition between states from one monitoring season to the next (t to t + 1) to estimate vital rates (recruitment and mortality). More details are available in @bischof2019rovquant and @bischof2020estimating . We used data augmentation [@royle2012augmentation], whereby additional, undetected individuals are available for inclusion in the population at each time step.

**Density and movement sub-model** We used a Bernoulli point process to model the distribution of individual activity centers. In the first year, individuals were located according to an intensity surface, which was a function of the locations of known packs at time t − 1 (see @bischof2019rovquant and @bischof2020estimating for more details). For all subsequent years (t \> 1), the location of individual activity centers was a function of the distance from previous activity centers (at time t − 1), as well as a function of the locations of known packs (at time t − 1). As in previous analyses [@milleret2022wolf; @milleret2023wolf], we used an exponential model to describe the movement of individuals between years, as it better accommodates distributions with long tails (i.e., a few individuals that make exceptionally long dispersal movements).

**Detection sub-model** SCR models take into account the spatial-variation in individual detection probability based on the distance between activity center locations (estimated by the density sub-model) and a given detector. A half-normal function was used to express the declining probability of detection with increasing distance between the activity center and the detector. In Scandinavia, DNA material from live wolves are collected following two main processes. First, authorities collect genetic samples and record the corresponding search effort during official searches ("structured sampling" thereafter). Second, DNA material can be collected by any member of the public (e.g., hunters) or by the authorities in a more or less opportunistic manner and associated search effort is not directly available ("unstructured" sampling thereafter). Currently, it is not possible to unambiguously distinguish between samples collected by the authorities during the structured or unstructured sampling in Rovbase. We therefore assigned each sample to structured or unstructured sampling based on whether a given sample matched in time and space with recorded search tracks: a sample was assigned to the "structured" sampling if it was collected by the authorities (marked as collected by "Statsforvalteren" or "SNO" in Rovbase) and if it was located within 500 m from a search track recorded on the same day. All remaining samples were assigned to the unstructured sampling. We assumed that both sampling processes could in theory occur within the entire study area and therefore used the same 10 km detector grid for both observation processes. Samples were then assigned to the closest detector (see details in @bischof2019rovquant and @bischof2020estimating ). However, spatial and temporal variation in the probability to detect a sample during the structured or unstructured sampling were assumed to be driven by different processes.

We accounted for spatio-temporal heterogeneity in detectability during structured sampling using:

-   Spatial-temporal variation in effort using the length of GPS search tracks in each detector grid cell.

-   Spatio-temporal variation in snow cover.

-   Spatio-temporal variation in monitoring regimes between jurisdictions (groups of counties in Sweden, monitoring regions in Norway).

-   Individual variation linked with the state of the individual (i.e., adult scent-marking individuals, other individuals).

-   Individual variation linked with a previous detection that could be expected to positively or negatively influence the probability of being detected at subsequent occasions, depending on the state of knowledge in the territory or neighboring territories.

We accounted for spatio-temporal heterogeneity in detectability during unstructured sampling using:

-   Spatio-temporal variation in unstructured sampling or carnivore observations (Figure A.1). For each detector grid cell and during each monitoring season (Oct 1 – Mar 31), we identified whether a) any carnivore sample had been registered in Rovbase, or b) any observation of carnivores had been registered in Skandobs. Roughly, the variable distinguishes areas with very low detection probability from those with some reasonable probability that carnivore DNA samples, if present in a detector grid cell, could have been detected and submitted for genetic analysis (Figure A.1).

-   Spatio-temporal variation in snow cover.

-   Spatial variation in accessibility measured as the average distance to the nearest road.

-   Spatio-temporal variation between countries.

-   Individual and temporal variation linked with the state of the individual (i.e., adult scent marking individuals, other individuals).

-   Individual and temporal variation linked with a previous detection that could be expected to positively influence the probability of being detected at subsequent occasions.

The different model components and data sources for covariates are described in detail in @bischof2019wolf; @bischof2019rovquant and @bischof2020estimating . Model fitting We fitted sex-specific Bayesian OPSCR models using Markov chain Monte Carlo (MCMC) simulation with NIMBLE version 0.12.2 [@turek2021efficient; @deValpine2017; @valpine2022] and nimbleSCR [@nimbleSCR2021] in R version 4.1.0 [@R2022] . We ran 4 chains each with 30 000 iterations, including a 10 000-iterations burn-in period. Due to the computing challenge associated with post-processing large amounts of data, we thinned chains by a factor of 10 from which abundance estimates were derived. We considered models as converged when the Gelman-Rubin diagnostics (Rhat, @gelman1992) was ≤1.1 for all parameters and by visually inspecting the trace plots.

**Abundance estimates** To obtain an estimate of abundance for any given area, we summed the number of predicted AC locations (live individuals) that fell within that area for each iteration of the MCMC chains, thus generating a posterior distribution of abundance for that area. In this fashion, abundance estimates and the associated uncertainty can be extracted for any spatial unit, including countries or counties (Figure A.2). Individuals detected near a border can have their model-predicted AC placed on different sides of that border in different model iterations (even if detections are only made on one side of the border). As a result, the probability of designating such individuals to either side of the border can be integrated into jurisdiction-specific abundance estimates. This is especially relevant for wolves detected along the Swedish and Norwegian border ("cross-boundary wolves", @wabakken2022 ); individual wolves can be partially designated to both countries [@bischof2016]. To ensure that abundance estimates for spatial sub-units (jurisdictions) add up to the overall abundance estimate, we used the mean and associated 95% credible interval limits to summarize posterior distributions of abundance. Combined (female/male) parameter estimates were obtained by merging posterior samples from the sex-specific models.

**Density maps** We used both the distribution of model-estimated AC positions and the scale parameter (σ) of the detection function to construct density maps based on individual utilization distributions. These maps are not only based on the position of the center of an individual’s home range, but also take into account the area over which that individual’s activity is spread, i.e., its space use [@bischof2020estimating]. To do so, we constructed raster maps (5 km resolution) of individual utilization distributions, scaled values in each raster to sum to one, and then summed rasters across individuals to create a single population-level raster map for each iteration. An overall density map was derived by calculating the mean across iterations in each cell [@bischof2020estimating].

**Other derived parameters** The average proportion of individuals detected and the associated uncertainty were obtained by dividing the number of individuals detected through NGS sampling (Table A.3) by the abundance estimates and their associated credible intervals, respectively. We derived the proportion of females in the population and the associated uncertainty by dividing the posterior distribution representing the number of females by the combined abundance estimate for males and females (Table A.3). Yearly population growth rates were calculated as λt = Nt+1/Nt (Table A.6).

**Focus on uncertainty** Although we reported median (or mean for abundance; see above) estimates for all parameters in the tables, we intentionally focused the main results of this report on the 95% credible interval limits of the estimates. We did so with the aim of drawing the reader’s attention to the uncertainty around population size estimates, rather than a single point estimate [@milleret2022wolf].

------------------------------------------------------------------------

# Results

## Non-invasive genetic samples and dead recoveries

A total of 20 171 (8 828 female; 11 343 male) genotyped `r engSpecies` genetic samples were included in the analysis, of which 76% originated from Sweden. These samples were associated with 1 838 (846 female; 992 male) individuals. We did not include individuals with unknown sex in this analysis. Among all genotyped samples, 8 803 (3660 female; 5143 male) were assigned to the structured sampling and 11 368 (5168 female; 6200 male) to the unstructured sampling. We also used 721 (324 female; 397 male) dead recoveries of wolves in the OPSCR model, of which 601 (267 female; 334 male) were due to legal culling and 120 (57 female; 63 male) due to other causes of mortality. The majority of dead recoveries (68%) originated from Sweden. Annual total and country-specific tallies of NGS samples and associated individuals, as well as dead recoveries included in the analysis are provided in the Appendices (NGS samples: Table A.2 and Table A.1, NGS individuals: Table A.3, dead recoveries: Table A.4)

## Density and abundance


Within its primary range (`r detectors$n.detectors*(detectors$resolution/1000)^2` $km^{2}$), the `r engSpecies` population was likely (95% credible interval) made up of between `r CI[1]` and `r CI[2]` individuals in `r years[n.years]`, with an estimated `r CI_F[1]` to `r CI_F[2]` females and `r CI_M[1]` to `r CI_M[2]` males.

`r EngSpecies` abundance for the entire study area (`r detectors$n.detectors*(detectors$resolution/1000)^2` $km^{2}$, excluding the buffer area) was likely (95% credible interval) between  `r CI[1]` and `r CI[2]` individuals in `r years[n.years]` (Table 1, Figure 1). Estimates refer to the status of the population at the start of the annual sampling period (Oct. 1). The proportion of females in the Scandinavian `r engSpecies` population was likely between 48% and 54% in 2023/2024. Based on the model-predicted location of ACs, we estimated that in 2023/2024, between 353 to 403 individuals could be attributed to Sweden and 56 to 73 to Norway (Table 1). See Table 1 for total and sex-specific abundance estimates for each country and management area. See Table A.5 for annual `r engSpecies` abundance estimates for all of Scandinavia and by region between 2014/2015 and 2023/2024. During the past three years, the `r engSpecies` population experienced a steady decrease from 494-525 in 2021/2022 to 414-470 in 2023/2024 (Figure 2, Table A.5). Note that estimates for different years (Figure 2) shown here differ slightly from those provided in Bischof et al. (2020), Milleret et al. (2021), Milleret et al. (2022c), and Milleret et al. (2023). This is due to the use of an updated OPSCR model and the inclusion of additional years of data. The analysis yielded annual density maps, which illustrate changes in the distribution of wolves over time (Figure A.3).

Table 1: Wolf population size estimates in 2023/2024 within the study area by sex at several spatial scales: the entire study area, by country, by management unit (carnivore management regions in Norway and "Rovdjursförvaltningsområden" in Sweden), and counties (“Län” in Sweden); see also Figure A.2. Only counties and management units that are within or that intersect the study area are included in the table. Readers should focus on the 95% credible interval provided in parentheses, as these - unlike mean values - convey uncertainty inherent in abundance estimates. Numbers are based on estimated activity center locations of wolves. Combined female-male estimates were obtained by joining sex-specific posterior distributions. Rounding may result in small deviations between total estimates and the sum of the estimates for constituent regions. Note that the numbers reported here are predictions from a statistical model which always represents an oversimplification of reality and is based on available data (NGS and dead recoveries). As a consequence, especially at the local scale, the model-estimated number of wolves based on DNA sampling can deviate from the number of wolves inferred from ancillary observations (e.g., camera traps).

# ```{r, Fig1, include = TRUE, fig.align = "center", fig.cap = paste0("Figure 1: ", species, " density in Scandinavia in ", years[length(years)], " based on individual utilization distributions throughout the study area (white background).", echo = FALSE}
# knitr::include_graphics(file.path(working.dir, "figures/UD_Density_LastYear.png"))
# ```

```{r, Fig2, include = TRUE, fig.align = "center", fig.cap = "Figure 2: Wolf population size estimates within the study area by country (blue: Sweden, red: Norway), and total (black) between 2014/2015 and 2023/2024. Darker and lighter bars show the 50% and 95% credible intervals, respectively. Credible intervals indicate uncertainty in estimates given the model and data used to generate the estimates. Light grey bars in the background show the total number of individuals detected within the entire study area. Changes in the model and the data can result in different estimates and associated uncertainty compared with estimates provided in previous reports by RovQuant. Note the higher precision (lower uncertainty) in the three monitoring seasons (2016/17, 2017/18, 2019/20) with higher sampling intensity in Sweden.", echo = FALSE}
knitr::include_graphics(file.path(working.dir, "figures/Abundance_TimeSeries.png"))
```

```{r, density timeSeries, fig.align = "center", fig.cap = paste0("Figure 3: ", species, " density in Norway between ", years[1], " and ", years[length(years)], "."), echo = FALSE}
knitr::include_graphics(file.path(working.dir, "figures/UD_Density_TimeSeries.png"))
```

## Vital rates

The OPSCR model produced annual estimates of legal hunting mortality, mortality associated with all other causes, and per capita recruitment rates (Figure 3; Table A.7). There was temporal variation in mortality rates, with the risk of mortality from causes other than culling generally higher than the risk of mortality from culling. Overall, scent-marking adult individuals had a higher survival compared to other individuals, with no pronounced systematic difference between sexes (Figure A.4).

Figure 3: Mortality probabilities due to legal culling (light green) and all other causes (dark green) for female and male wolves. Darker and lighter bars show the 50% and 95% credible intervals, respectively. Shown are overall estimates throughout the study area. Estimates refer to deaths occurring between the start of one sampling season and the start of the next.

Figure 4: Estimated annual number of recruits for female and male wolves. Recruitment represents the number of new individuals present in the population on Oct 1 (i.e., individuals that were born between the two consecutive monitoring seasons and survived to Oct 1 or that immigrated in the study area). Darker and lighter bars show the 50% and 95% credible intervals, respectively.

## Detection probability

The overall proportion of detected individuals in the study area was likely between 77% and 88% in 2023/2024 (Table A.11). The baseline detection probability for the structured and unstructured sampling varied both in time and space (Figure A.5 and Figure A.6). More specifically, the length of recorded search tracks positively affected detection probability of the structured sampling (2023/2024; males: β2Structured =0.17, CrI: 0.12 - 0.22; females: β2Structured = 0.28, CrI: 0.19 - 0.38; Table A.9). However, the previous detection of individuals and the average proportion of snow cover had no strong effect on detection probability for the structured sampling (Table A.9). Unstructured search effort derived using the observation data in Skandobs and Rovbase had a strong positive effect on detection probability during unstructured sampling (2023/2024; males: β4Unstructured = 1.35, CrI: 0.78 - 2.06; females: β4Unstructured = 1.67, CrI: 1.08 - 2.34; Table A.10). The effect of previous detection also tended to have a positive effect on detection probability during unstructured sampling, but the pattern was not consistent across years (Table A.10).

------------------------------------------------------------------------

# Suggestions for future improvements

RovQuant continues to work on improving the functionality and efficiency of OPSCR models. We recommend the following developments for future analyses of the Scandinavian wolf monitoring data:

1.  Review and adjust spatial covariates on density. This may involve the addition of land cover and topographic variables.

2.  Account for individual heterogeneity in detectability/space use, for example by using a finite-mixture approach [@cubaynes2010importance] .

3.  Consider alternative detection models that do not assume a half-normal shape and/or circular home ranges [@sutherland2015].

4.  Account for spatial variation in vital rates (i.e., survival, recruitment; @milleret2022estimating ).

------------------------------------------------------------------------

# Acknowledgements

This work was made possible by the large carnivore monitoring programs and the extensive monitoring and observation data collected by Swedish (Länstyrelsena) and Norwegian (SNO) wildlife management authorities, as well as the public in both countries. Our analysis relied on genetic analyses conducted by the laboratory personnel at the DNA laboratories at the Swedish University of Agricultural Sciences, and the Norwegian Institute for Nature Research. We also thank Swedish and Norwegian wildlife managers for feedback provided during project RovQuant and the Research Council of Norway for partial funding (NFR 286886: project WildMap and NFR 345279: project PopFlow). 

------------------------------------------------------------------------

# Data availability

Data, R code to reproduce the analysis, as well as figures, tables, and raster maps (Figure A.3) are available at <https://github.com/richbi/RovQuantPublic>.

------------------------------------------------------------------------

# References

::: {#refs}
:::

------------------------------------------------------------------------

# Metadata

```{r, metadata, echo = F}
utils::sessionInfo()
```
